### PR TITLE
add server side logs ;)

### PR DESCRIPTION
**Pull request by UmbraPlus**
## Summary
Added new log within server side code that will print out a additional log on the server side request.

## Testing
Run the program with the following commands:
```minikube start```

and then this command next:
```skaffold dev```

Once the program starts running, you should now see it display two sets of logs, one from the server and one from the client.
image

### CODE REVIEW

File: server/bin/.project
Comment (wcygan) [inline server/bin/.project:1]: `.gitignore` this file

File: server/bin/.settings/org.eclipse.core.resources.prefs
Comment (wcygan) [inline server/bin/.settings/org.eclipse.core.resources.prefs:1]: `.gitignore` this file

File: server/bin/.settings/org.eclipse.m2e.core.prefs
Comment (wcygan) [inline server/bin/.settings/org.eclipse.m2e.core.prefs:1]: `.gitignore` this file

File: server/bin/pom.xml
Comment (wcygan) [inline server/bin/pom.xml:1]: `.gitignore` this file

File: server/bin/server.yaml
Comment (wcygan) [inline server/bin/server.yaml:1]: `.gitignore` this file

File: server/bin/skaffold.yaml
Comment (wcygan) [inline server/bin/skaffold.yaml:1]: `.gitignore` this file

File: server/bin/src/main/java/com/example/server/PingServer.class
Comment (wcygan) [inline server/bin/src/main/java/com/example/server/PingServer.class:1]: `.gitignore` this file

File: server/bin/src/main/java/com/example/server/PingServer$PingServiceImpl.class
Comment (wcygan) [inline server/bin/src/main/java/com/example/server/PingServer$PingServiceImpl.class:1]: `.gitignore` this file

**Review by wcygan — CHANGES_REQUESTED** [review]
Java code looks fine, but you're including too much in the PR.

Please figure out how to add those extra files to `.gitignore`

File: server/bin/server.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: ping-server-service
+spec:
+  selector:
+    app: ping-server
+  ports:
+    - protocol: TCP
```
Comment (Faro0q) [inline server/bin/server.yaml]: No need to add protocol since it defaults to TCP

File: server/bin/.project
Comment (UmbraPlus) [reply server/bin/.project:1]: Done for all the listed files below

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
\ No newline at end of file
+**/.DS_Store
+server/bin/.project
+server/bin/.settings/org.eclipse.core.resources.prefs
+server/bin/.settings/org.eclipse.m2e.core.prefs
+server/bin/pom.xml
+server/bin/server.yaml
+server/bin/skaffold.yaml
+server/bin/src/main/java/com/example/server/PingServer$PingServiceImpl.class
+server/bin/src/main/java/com/example/server/PingServer.class
```
Comment (wcygan) [inline .gitignore]: Please find a way to ignore all of these files using a single matching statement

**Review by wcygan — CHANGES_REQUESTED** [review]

**Review by Faro0q — CHANGES_REQUESTED** [review]

File: server/bin/server.yaml (hunk 1 above)
Comment (wcygan) [reply server/bin/server.yaml]: Very insightful, thank you @Faro0q. Maybe you can open a PR for the change?

File: .gitignore (hunk 2 above)
Comment (UmbraPlus) [reply .gitignore]: Done with the latest commit.

**Review by wcygan — APPROVED** [review]
LGTM

File: server/bin/src/main/java/com/example/server/PingServer$PingServiceImpl.class
Comment (wcygan) [inline server/bin/src/main/java/com/example/server/PingServer$PingServiceImpl.class:1]: This class is still not ignored?

**Review by wcygan — CHANGES_REQUESTED** [review]

File: server/bin/server.yaml (hunk 1 above)
Comment (Faro0q) [reply server/bin/server.yaml]: kys

Comment (Faro0q) [reply server/bin/server.yaml]: PR owner should be adjusting these changes.

File: server/bin/src/main/java/com/example/server/PingServer$PingServiceImpl.class
Comment (wcygan) [reply server/bin/src/main/java/com/example/server/PingServer$PingServiceImpl.class:1]: I was able to resolve with:

```bash
git ls-files
git rm --cached server/bin/src/main/java/com/example/server/PingServer\$PingServiceImpl.class
```

I also changed the `.gitignore` statement to be leaner

**Review by wcygan — APPROVED** [review]

**Pull request merged**